tests/core/audit_test.py

```diff
@@ -24,7 +24,12 @@ def test_no_baseline(self, mock_printer):
             assert mock_printer.message == ''
 
     def test_quit_before_making_decision(self, mock_printer):
-        with self.mock_env(['q']) as m:
+        with mock.patch(
```

Comment:
Let's throw this in `mock_env`. After all, we're already patching a bunch of things there.